src/client/pydaos/raw/daos_api.py

```diff
@@ -1374,12 +1409,25 @@ def __init__(self, context, cuuid=None, poh=None, coh=None):
         self.coh = ctypes.c_uint64(0)
         self.poh = ctypes.c_uint64(0)
         self.info = daos_cref.ContInfo()
+        self.cont_input_values = DaosContProperties()
+        self.cont_prop = None
+
+    def get_cont_prop(self):
+        """ Get the container properties """
+        return self.cont_input_values
```

Comment:
container.query should provide the container properties (info data). Maybe I should add a clear comment (or change the method name) stating the get_cont_prop and set_cont_prop are not really setting the  container properties.  The functions are basically used as a wrapper method to set various container properties during creation. Instead of having several input parameters to the create method, we are trying to pack the input params to a structure.

Reply:
I see. If those helpers are to manage properties, then they should not be methods of the container class. Maybe we could add them to the DaosContPropEnum? Otherwise, just standalone helpers would work too.

Reply:
Sure. I will add them to DaosContPropEnum. Also, changing the method name from 
set_cont_prop ---> set_con_create_params and get_cont_prop ---> get_con_create_params.
Added some more comments so user gets a better idea about the function usage.

Reply:
Done. Also remove the need for set input... Once the input structure is exported, user has to modify the values and pass them for create methods.